test/e2e-helm/e2e_helm_project_locally_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
File names are a little verbose IMO. WDYT about the following?
- `test/e2e-helm/e2e_helm_cluster_test.go`
- `test/e2e-helm/e2e_helm_local_test.go`